### DISCUSSION TITLE
View and React to gists

### DISCUSSION THREAD

**Discussion by Tarektouati** [community/community#3406] [Mobile] [answered]
When I click on gist links : it open's it in an webview.

It still can read the gist, etc ..., but I can't star, reply or react, to the gist, unless I login on the webview (which is a pain in ass).

**Comment by candyho** [comment, marked as answer]
Hey, thanks for sharing your feedback. We hear you! Feel free to upvote on this topic related to Gists support to help boost awareness of this request! - https://github.com/github/feedback/discussions/3331
